src/OpenTelemetry.Exporter.Console/CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ please check the latest changes
 ## Unreleased
 
 * Removes .NET Framework 4.5.2 support. The minimum .NET Framework version
```

Comment:
Add 4.6 as well

Reply:
we never had 4.6 for this console exporter.